src/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  flex-basis: 300px;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
```

Comment:
move as well

Reply:
do you mean font size?

Reply:
also was idea to move this property to tag selector `html` in css)

Reply:
done